Jamulus.pro

```diff
@@ -54,7 +54,14 @@ TRANSLATIONS = src/res/translation/translation_de_DE.ts \
     src/res/translation/translation_sv_SE.ts \
     src/res/translation/translation_zh_CN.ts
 
-INCLUDEPATH += src
+OBJECTS_DIR = obj
+
+MOC_DIR = src/Generated
```

Comment:
```suggestion
MOC_DIR = src/compiled
```

Reply:
compiled is not the correct word either, since these are generated header and cpp files, nothing compiled yet....

BTW "Generated" is the term used in the Qt documentation ["The C++ source file generated by moc"](https://doc.qt.io/archives/qt-4.8/moc.html)

Reply:
Ok. Then use "generated"